docs/development/testguide.rst

```diff
@@ -110,6 +212,18 @@ Astropy Test Function
 
 .. autofunction:: astropy.test
 
+pytest
+======
+
+The test suite can be run directly from the native ``pytest`` command. In this
```

Comment:
This actually isn't currently supported even with the `v2.0.x` backport of test updates. I'm not sure whether it's worth the effort to support, so I should probably just remove this section of the documentation.

Reply:
Sure, please remove it from the docs if it's not working atm. If it's not clear whether it's worth at all to support then I suggest to open an issue as a reminder.